git/compat.py

```diff
@@ -56,7 +56,9 @@ def safe_decode(s):
         return s
     elif isinstance(s, six.binary_type):
         if PRE_PY27:
-            return s.decode(defenc)  # we're screwed
+            # Python 2.6 does not support the `errors` argument, so we cannot
+            # control the replacement of unsafe chars in it.
+            return s.decode(defenc)
```

Comment:
No worries, I've been there. Maintaining OSS libraries while also getting work done is a hard balance to strike. On the plus side, patching/maintaining this project is important for our team now, and we're committed to only invest in it more. So if you'd like to have a contributor on board, I'd be more than happy to help out there!